### PR TITLE
[a11y] toolbar

### PR DESCRIPTION
**Pull request by brianteeman**
This PR adds the role=toolbar and the aria-label=toolbar to the standard toolbar used in 

- all admin components at the top
- with editor-xtd plugins below the textarea when tinmyce is not used
- the configuration menu options on the front end

To test view the generated source and you will now see something like
`<div class="btn-toolbar" role="toolbar" aria-label="toolbar">`

### CODE REVIEW

**Comment by infograf768** [comment]
Same question as for 4.0 : do we use toolbar in frontend?

File: layouts/joomla/toolbar/containeropen.php [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 defined('JPATH_BASE') or die;
 ?>
-<div class="btn-toolbar" id="<?php echo $displayData['id']; ?>">
+<div class="btn-toolbar" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR') ?>" id="<?php echo $displayData['id']; ?>">
```
Comment (Quy) [inline layouts/joomla/toolbar/containeropen.php]: Add `;` as the other echo statements have it.

Comment (brianteeman) [inline layouts/joomla/toolbar/containeropen.php]: good spot - thanks

File: components/com_config/view/config/tmpl/default.php [hunk 2]
```diff
@@ -29,7 +29,7 @@
 	<div class="row-fluid">
 		<!-- Begin Content -->
 
-		<div class="btn-toolbar">
+		<div class="btn-toolbar" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR') ?>">
```
Comment (Quy) [inline components/com_config/view/config/tmpl/default.php]: Add ;.

File: components/com_config/view/modules/tmpl/default.php [hunk 3]
```diff
@@ -44,7 +44,7 @@ class="form-validate">
 		<!-- Begin Content -->
 		<div class="span12">
 
-			<div class="btn-toolbar">
+			<div class="btn-toolbar" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR') ?>">
```
Comment (Quy) [inline components/com_config/view/modules/tmpl/default.php]: Add `;`.

File: components/com_config/view/templates/tmpl/default.php [hunk 4]
```diff
@@ -29,7 +29,7 @@
 	<div class="row-fluid">
 		<!-- Begin Content -->
 
-		<div class="btn-toolbar">
+		<div class="btn-toolbar" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR') ?>">
```
Comment (Quy) [inline components/com_config/view/templates/tmpl/default.php]: Add `;`.

File: layouts/joomla/editors/buttons.php [hunk 5]
```diff
@@ -12,7 +12,7 @@
 $buttons = $displayData;
 
 ?>
-<div id="editor-xtd-buttons" class="btn-toolbar pull-left">
+<div id="editor-xtd-buttons" class="btn-toolbar pull-left" role="toolbar" aria-label="<?php echo JText::_('JTOOLBAR') ?>">
```
Comment (Quy) [inline layouts/joomla/editors/buttons.php]: Add `;`.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fe84c850d34525f965e6d6b49b5d4cf86a2ec2a2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15585">issues.joomla.org/tracker/joomla-cms/15585</a>.</sub>

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on fe84c850d34525f965e6d6b49b5d4cf86a2ec2a2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15585">issues.joomla.org/tracker/joomla-cms/15585</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**